zulipterminal/ui_tools/boxes.py

```diff
@@ -167,6 +170,15 @@ def need_recipient_header(self) -> bool:
         else:
             raise RuntimeError("Invalid message type")
 
+    def _check_if_self_PM(self) -> bool:
+        recipient_list = self.message['display_recipient']
+        if len(recipient_list) == 1 and \
+            recipient_list[0]['email'] == \
+                self.model.user_email:
+            return True
+        else:
+            return False
```

Comment:
Whenever one returns a boolean, it's worth considering whether to return a value explicitly (as you do above) or returning the value of an expression; can you see how you could do the latter here? Would it be simpler?